src/jailer/src/env.rs

```diff
@@ -416,8 +416,13 @@ impl Env {
         // a new PathBuf, with something like chroot_dir.join(exec_file_name) ?!
         self.chroot_dir.push(exec_file_name);
 
-        // TODO: hard link instead of copy? This would save up disk space, but hard linking is
-        // not always possible :(
+        // We do a copy instead of a hard-link for 2 reasons
+        // 1. hard-linking may not be possible if the file is in another device
```

Comment:
Nit: I think it is strictly not possible to hardlink across filesystems.

Reply:
Fixed